src/Outline.ts

```diff
@@ -24,25 +24,6 @@ const defaultProps: Types.OutlineProps = {
 
 let _props: Types.OutlineProps = defaultProps;
 
-let _fullScreenEventName: string | undefined;
```

Comment:
this was a side effect and leaking into the bundle even when outline wasn't used